gitbook/tables/tuplets.html

```diff
@@ -2301,37 +2301,37 @@ <h2 id="recommended-stylistic-alternates">Recommended stylistic alternates</h2>
 <tbody>
 <tr>
 <td style="text-align:center"><span class="bravura_large">&#xF4CC;</span></td>
-<td><strong>uniE880.ss06</strong><br><em>tuplet0Light</em><br>Tuplet 0 (light)</td>
```

Comment:
These stylistic alternates appear as clef glyphs on my system (though they did not change in this pull request, I don't think I've viewed this page since they were added).